cypress/support/api/requests/Checkout.js

```diff
@@ -280,3 +280,25 @@ export function getCheckout(token) {
   }`;
   return cy.sendRequestWithQuery(query).its("body.data.checkout");
 }
+
+export function orderCreateFromCheckout(
+  checkoutId,
+  removeCheckoutFlag = true,
```

Comment:
default parameters should be last. When it will be last, you could invoke this function without passing removeCheckoutFlag - orderCreateFromCheckout(checkout.id, token)